react-native-libraries.json

```
@@ -9294,5 +9294,12 @@
       "https://github.com/Splicer97/react-native-osmdroid/tree/main/example"
     ],
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/sparkfabrik/sparkfabrik-react-native-idfa-aaid",
+    "npmPkg": "@sparkfabrik/react-native-idfa-aaid",
+    "ios": true,
+    "android": true,
+    "expo": true
```

```suggestion
    "android": true
```

Expo flag means that the library will work with Expo Go app (every library can work with `dev-client`s), and after the quick glance at the package source, it probably won't work. It would be nice to confirm that, or just remove the `expo` flag for now.

---

@Simek sorry I didn't check, I was thinking just "it will work with an Expo app" which is true because I use it also in an Expo app but now I get it, changed the line!